cpp/test/eigen_solvers.cu

```diff
@@ -75,7 +82,12 @@ TEST(Raft, SpectralSolvers)
   using value_type = double;
 
   handle_t h;
-  ASSERT_EQ(0, h.get_device());
+  ASSERT_EQ(0,
```

Comment:
Same as above

Reply:
Ooh, this one is strange. I'm not sure why clang-format did this.